nservicebus/operations/metrics-service-control.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Metrics data to ServiceControl
+summary: Reporting metrics to ServiceControl.
+reviewed: 2017-09-26
+component: MetricsServiceControl
+related:
+ - nservicebus/operations
+related:
+ - samples/logging/metrics
+---
+
+
+This component enables sending monitoring data to ServiceControl.Monitoring instance.
+
+
+## Enabling
+
+It can be enabled via:
+
+snippet: SendMetricDataToServiceControl
+
+
+#### Service Control Metrics Address:
+
+The default is "particular.monitoring" which is ServiceControl monitoring instance input queue.
+
+
+#### Instance Id:
+
+An override for `$.diagnostics.hostid` and `$.diagnostics.hostdisplayname`.
+
+By default the monitoring plug-in will use [host identifiers](/nservicebus/hosting/override-hostid.md) to identify the monitored endpoint instance.
+
+The value of `$.diagnostics.hostdisplayname` is expected to be human readable. This might be problematic when using the [Azure Host](/nservicebus/hosting/cloud-services-host/faq.md#host-identifier) as it uses Role Name for the `$.diagnostics.hostdisplayname`, which results in the same display name for all instances.
```

Comment:
given the cloud-services-host is becoming obsolete i dont think that is a good place to send people for more information on azure hosting

Reply:
I agree it's not perfect. But `instanceId` exists pretty much only because of this :/.

Reply:
but people can host in a cloud-services without using cloud-services-host. so you are saying in that case they do not need the instanceid?

Reply:
Yes. The only scenario that required this argument I know of is CloudService host.

Reply:
There is another use for that, for deployment reasons as described here: https://github.com/Particular/ServiceControl/issues/329

Reply:
WebJobs would have a similar problem. Scale happens automatically and instance ids are not deterministic. [Link to GG](https://groups.google.com/forum/#!searchin/particularsoftware/webjob$20host$20id%7Csort:relevance/particularsoftware/URnuGDf27QM/TCPsGxD9AAAJ)

Reply:
@SeanFeldman just to validate my assumptions. I though that Cloud Service host was the only problematic piece as it was doing a custom HostId, HostDisplayName override at the header level. 

My understanding was that even if I host using `WebJobs` I could use [host override api](https://docs.particular.net/nservicebus/hosting/override-hostid?version=core_7#overriding-the-host-identifier) and set:
 * `HostId` to deterministic GUID
 * `HostDisplayName` to some human readable thing based on rolename

The problem with CloudService host as I understood it was that it would override whatever I would set using custom hostid api.

Reply:
Not entirely sure. @yvesgoeleven do you recall if that's what CS host did?

Reply:
Seems the original concern here hasn't been addressed, is that intentional or fell through the cracks?

Reply:
We have dropped any reference to `NSerfviceBus.Hosthost` and only included the relevant Azure Host related info